src/CookieRequest.jl

```diff
@@ -5,12 +5,12 @@ import ..Layer, ..request
 using URIs
 using ..Cookies
 using ..Pairs: getkv, setkv
-import ..@debug, ..DEBUG_LEVEL
+import ..@debug, ..DEBUG_LEVEL, ..access_threaded
 
-const default_cookiejar = [Dict{String, Set{Cookie}}()]
+const default_cookiejar = Dict{String, Set{Cookie}}[]
 
 function __init__()
-    Threads.resize_nthreads!(default_cookiejar)
+    resize!(empty!(default_cookiejar), Threads.nthreads())
```

Comment:
curious why the `empty!` call here?

Reply:
Just in case some garbage is left in it from the precompilation process that ends up getting serialized.